pkg/cloudprovider/servicecontroller/servicecontroller.go

```diff
@@ -217,6 +227,7 @@ func (s *ServiceController) processDelta(delta *cache.Delta) (error, bool) {
 	case cache.Deleted:
 		err := s.balancer.EnsureTCPLoadBalancerDeleted(s.loadBalancerName(service), s.zone.Region)
 		if err != nil {
+			s.eventRecorder.Event(service, "delete balancer failed", err.Error())
```

Comment:
Another nit: delete -> deleting for symmetry with the creating failure-event (or change that one from creating -> create)